blocks/api/factory.js

```diff
@@ -71,15 +71,17 @@ export function createBlock( name, blockAttributes = {}, innerBlocks = [] ) {
  *
  * @return {Object} A cloned block.
  */
-export function cloneBlock( block, mergeAttributes = {}, innerBlocks = block.innerBlocks ) {
+export function cloneBlock( block, mergeAttributes = {}, innerBlocks ) {
```

Comment:
Could we just keep the default value and just continue with the `Array#map` clone, avoiding the truthiness checking below?

Reply:
Hi @aduth, if we do this change even when innerBlocks are explicitly passed, we will clone them. Clonning them will change their uid's, when I call cloneBlocks and I explicitly pass the innerBlocks array I expect that the uids I passed are respected that's why I changed this logic to not clone the blocks if they are explicitly passed.